jetty-server/src/main/config/modules/server.mod

```diff
@@ -62,9 +62,12 @@ patch-module: servlet.api=lib/jetty-schemas-3.1.jar
 ## Maximum number of error dispatches to prevent looping
 # jetty.httpConfig.maxErrorDispatches=10
 
-## Cookie compliance mode of: RFC2965, RFC6265
+## Cookie compliance mode for parsing: RFC2965, RFC6265
 # jetty.httpConfig.cookieCompliance=RFC6265
 
+## Cookie compliance mode for generating: RFC2965, RFC6265
+# jetty.httpConfig.setCookieCompliance=RFC6265
```

Comment:
the only difference between this and the one above is `set`??
I'm confused?

Why would you want two different CookieCompliance settings here?

Reply:
Ugh, we need a different name, this is going to be so confusing and everyone will forget in 2 months.
How about `parserCookieCompliance` and `generatorCookieCompliance`?
And deprecate `cookieCompliance` in favor of `parsingCookieCompliance`.

Reply:
@gregw you got 2 of us confused. There is no Javadoc in the `*.mod` file.
What harm can it be to be more explicit and add `parser` and `generator` prefixes to be crystal clear?

Reply:
@sbordet @joakime How's that with improved documentation?

Reply:
@sbordet perhaps instead of `parser<name>` and `generator<name>` it should be `requestCookie` and `responseSetCookie`?